tests/unit/decoder/src/test_decode_manifest.c

```diff
@@ -365,15 +365,93 @@ static uint8_t cbor_envelope_with_6_components[] = {
 			0x40, /* bytes(0) */
 
 	0x03, /* suit-manifest */
-	0x58, 0x1c, /* bytes(28) */
+	0x58, 0x2f, /* bytes(37) */
 	0xa3, /* map (3 elements) */
 	0x01, /* suit-manifest-version */ 0x01,
 	0x02, /* suit-manifest-sequence-number */ 0x10,
 	0x03, /* suit-common */
-		0x55, /* bytes(21) */
+		0x58, 0x27, /* bytes(30) */
 		0xA1, /* map (1 element) */
 			0x02, /* suit-components */
-				0x86, /* array (6 elements) */
+				0x8c, /* array (13 elements) */
```

Comment:
The vector says about `12_components`, but the array contains `13 elements`?

Reply:
This test vector is meant to have 12 components, which is just enough to fit. See below that I have added a new test that is supposed to succeed parsing.

The next test vector has 13 components and is meant to fail.